common/OpTestHMC.py

```diff
@@ -576,6 +808,12 @@ def close(self):
         log.debug("HMC close -> TERMINATE")
 
     def connect(self, logger=None):
+        '''
+        Connects HMC console
```

Comment:
This is actualy connecting to lpar console, so please update the statement as per functionality